packages/web-components/tests/e2e-storybook/cypress/integration/masthead/masthead-custom-search.e2e.js

```diff
@@ -28,30 +35,30 @@ describe('cds-masthead | custom search (desktop)', () => {
   });
 
   it('should open the search bar on click', () => {
-    cy.get('cds-masthead > cds-search-with-typeahead')
+    cy.get(`${_selectors.masthead} > ${_selectors.mastheadSearch}`)
       .shadow()
-      .find('.bx--header__search--search')
+      .find(_selectors.mastheadSearchButton)
       .click();
 
     cy.takeSnapshots();
   });
 
-  xit('should display grouped results with hrefs', () => {
+  it('should display grouped results with hrefs', () => {
     // Mock grouped search typeahead API. Below we user the "cloud" search
     // string. Every keypress will trigger an API request, so here we mock each
     // successive cumulative search query.
-    [('c', 'cl', 'clo', 'clou', 'cloud')].forEach(query => {
+    ['c', 'cl', 'clo', 'clou', 'cloud'].forEach(query => {
       cy.intercept(`https://ibmdocs-dev.dcs.ibm.com/docs/api/v1/suggest?query=${query}&lang=undefined&categories=&limit=6`, {
```

Comment:
I'm finding that this is the only test failing locally. It appears to be failing based on CORS issues (see below). The server being requested appears to have changed, so we probably need to update the URL here to match, so that we're mocking it properly.

![image](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/assets/191049/2fb60cc7-08df-45bb-a0f5-e41ce6eed622)